### DISCUSSION TITLE
Level always at 0% in heating groups after last update

### DISCUSSION THREAD

**Discussion by ITCrowdIOB** [OpenCCU/OpenCCU#2694] [🇺🇸 Q&A]
### Describe the issue you are experiencing

The data point Level in a device group always shows 0%. The groups consist of two or three radiator thermostats with a wall thermostat. The error occurs here since the update to 3.75.6.20240316. The data point (Level) on the single device works.

### Describe the behavior you expected

Before the update, the last changing value was always displayed here. 

### Steps to reproduce the issue

Change in value of the valve position.

### What is the version this bug report is based on?

3.75.6.20240316.

### Which base platform are you running?

rpi4 (RaspberryPi4, ARM64/aarch64)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
N/A
```

**Comment by jens-maus** [comment]
Sorry, but your description is too generic. No technical details on the actual homematic devices you are using in the group, etc. In additon, this cannot be a general issue/problem since thousands of other users are happily using it. Thus, will convert it to a discussion so that you seek for help.
